mypy_django_plugin/transformers/managers.py

```diff
@@ -274,6 +285,19 @@ def resolve_manager_method(ctx: AttributeContext) -> MypyType:
 
     if isinstance(ctx.type, Instance):
         return resolve_manager_method_from_instance(instance=ctx.type, method_name=method_name, ctx=ctx)
+    elif isinstance(ctx.type, UnionType):
+        if any(isinstance(instance, AnyType) for instance in ctx.type.items):
+            ctx.api.fail(
+                f'Unable to resolve return type of queryset/manager method "{method_name}" on a union involving Any',
+                ctx.context,
+            )
+            return AnyType(TypeOfAny.from_error)
+        else:
+            resolved = [
+                resolve_manager_method_from_instance(instance=instance, method_name=method_name, ctx=ctx)
+                for instance in ctx.type.items
+            ]
+            return get_proper_type(UnionType(tuple(resolved)))
```

Comment:
```suggestion
            resolved = tuple(
                resolve_manager_method_from_instance(instance=instance, method_name=method_name, ctx=ctx)
                for instance in ctx.type.items
            )
            return get_proper_type(UnionType(resolved))
```

We can build a tuple directly instead of casting

Reply:
ah yeah, reasonable, also I fixed the broken tests

Reply:
fixed